packages/slate/test/interfaces/Editor/positions/all/block-multiple-inline-multiple-reverse.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsx jsx */
```

Comment:
I'm not sure about the naming for these new test files. Alternate suggestions would very much be welcomed.

I feel the names are getting out of hand in this case, but I wasn't sure how to reconcile the existing pattern with these compound criteria.